### PR TITLE
Add ELK stack roles to devstack

### PR DESCRIPTION
**Pull request by rachelg4**
They can be disabled using the ENABLE_ELK var

### CODE REVIEW

**Comment by rachelg4** [comment]
@stvstnfrd @caesar2164

File: playbooks/vagrant-devstack.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@
     edxapp_use_custom_theme: true
     edxapp_theme_name: 'lagunita'
     edxapp_theme_source_repo: 'https://github.com/Stanford-Online/lagunita-theme.git'
+    ENABLE_ELK: true
```
Comment (caesar2164) [inline playbooks/vagrant-devstack.yml]: shouldn't this be configurable?

Comment (stvstnfrd) [inline playbooks/vagrant-devstack.yml]: @caesar2164 It's literally a variable to configure it ;)

We have to default it to something;
you disable it by setting `ENABLE_ELK=false`.

Comment (caesar2164) [inline playbooks/vagrant-devstack.yml]: oh doh!

I was expecting to see that pattern that pulls from secrets, nvmd!

**Comment by caesar2164** [comment]
👍

**Comment by stvstnfrd** [comment]
This was merged in d0668d90d995c4eb159985226cefe198625dd419.

Github lost track of the commit because I edited it (to remove trailing whitespace) prior to merging.

**Pull request closed without merging**